.github/workflows/solidity.yml

```diff
@@ -29,6 +29,34 @@ jobs:
       - name: Run solhint
         run: ./tools/ci/solhint
 
+  solidity-coverage:
+    name: Solidity Coverage
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+      - name: Yarn cache
+        uses: actions/cache@v2
+        env:
+          cache-name: yarn-cache
+        with:
+          path: |
+            ~/.npm
+            ~/.cache
+            **/node_modules
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - run: yarn install --frozen-lockfile
+      - name: Run coverage
+        run: ./tools/ci/solidity_coverage
```

Comment:
The coverage script is run here to make sure no errors occur, but we don't do anything with the percentage outputs at the moment.